qa/index.html

```diff
@@ -37,8 +37,26 @@
           );
           const doc = libreoffice.loadDocument(uri);
           doc.on("ready", (x) => {
-            console.log(x);
+            console.log("Document Ready", x);
+
+            const encodedData = new TextEncoder().encode(
+              "testing 123, will is cool"
+            );
+
+            const clipboardData = [
+              {
+                mimeType: "text/plain;charset=utf-8",
```

Comment:
Should be changed to just text/plain, as we discovered yesterday.

Maybe should add to the TS docs the weird behavior for setClipboard/getClipboard.

Reply:
Good calls